randovania/games/dread/json_data/Burenia.txt

```diff
@@ -2591,6 +2342,8 @@ Extra - asset_id: collision_camera_021
 
 > Shaft Base; Heals? False
   * Layers: default
+  > Pickup (Missile+ Tank)
+      Screw Attack
```

Comment:
Should probably connect Shaft Base and the Door to Nav Station South in both directions. Currently this forces collecting the Missile+ in either direction.